### PR TITLE
Janitor Edits

### PR DESCRIPTION
**Pull request by Mark9013100**
* Adds a second janitor slot, as discussed here: http://ss13polaris.com/forums/viewtopic.php?f=18&t=662

* Adds the janitorial belt, can hold a small list of items. Credit goes towards Steelpoint.

* Doubles the amount of items in the custodial wardrobe and Janitorial level-three biohazard closet.

* Increases the amount of janitorial gear in the janitorial supply crate.

* Adds the 'Custodian' alt-title.

Map edits screenshot:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 1]
```diff
@@ -67,18 +67,34 @@
...
 	new /obj/item/weapon/caution(src)
 	new /obj/item/weapon/caution(src)
 	new /obj/item/weapon/caution(src)
 	new /obj/item/device/lightreplacer(src)
+	new /obj/item/device/lightreplacer(src)
+	new /obj/item/weapon/storage/bag/trash(src)
 	new /obj/item/weapon/storage/bag/trash(src)
+	new /obj/item/weapon/storage/belt/janitor(src)
+	new /obj/item/weapon/storage/belt/janitor(src)
+	new /obj/item/clothing/shoes/galoshes(src)
```
Comment (PsiOmegaDelta) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: The room is a bit awkwardly shaped but it seems like a better idea to just map in one more of each of the changed closets. Just my opinion tho.
Lockers do magically adjust their size according to their contents (on spawn) if there's too much but I've always felt that was a bit of a hack.

Comment (Mark9013100) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: Sounds reasonable.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -233,3 +233,26 @@
...
+	storage_slots = 7
+	max_w_class = 3
+	can_hold = list(
+		/obj/item/clothing/glasses,
+		/obj/item/device/flashlight,
+		/obj/item/weapon/grenade,
+		/obj/item/device/pda,
+		/obj/item/device/radio/headset,
+		/obj/item/clothing/gloves,
+		/obj/item/clothing/mask/surgical, //sterile mask
```
Comment (Kelenius) [inline code/game/objects/items/weapons/storage/belt.dm]: Due to how the indentation checker works, you need to put a comma at the end of the comment here to pass Travis checks.

Comment (Mark9013100) [inline code/game/objects/items/weapons/storage/belt.dm]: Didn't know that, thanks!

**Comment by Datraen** [comment]
While I'm in agreement with Psi, the PR as a whole isn't a problem.
So, :+1:

**Pull request merged**